pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,16 @@
+# 8.1.0
+2025-04-02 (Date of Last Commit)
+
```

Comment:
Implemented a unified STARsolo execution strategy to ensure consistent and accurate cell barcode correction across the entire dataset. This update resolves discrepancies that previously arose from sharded (partitioned) processing, where each shard independently corrected barcodes using incomplete local priors. By consolidating barcode frequency calculations and applying correction globally, the pipeline now mirrors the behavior of DropSeq and Cell Ranger.